submissions/BlueLamantine/js-dom/dom.js

```diff
@@ -0,0 +1,154 @@
+const COFFEE = [{
+	title: 'Doppio',
+	id: 'doppio',
+	classes: {
+		bgColor: 'doppio-bg',
+		textColor: 'latte-text'
+	},
+	paragraph: 'Double shot of espresso. Straight.',
+	state: true
+}, {
+	title: 'Americano',
+	id: 'americano',
+	classes: {
+		bgColor: 'americano-bg',
+		textColor: 'cappuccino-text'
+	},
+	paragraph: 'Shots of espresso diluted with water.',
+}, {
+	title: 'Macchiato',
+	id: 'macchiato',
+	classes: {
+		bgColor: 'macchiato-bg',
+		textColor: 'cappuccino-text'
+	},
+	paragraph: 'Espresso with a dash of frothy foamed milk.',
+}, {
+	title: 'Flat white',
+	id: 'flatw',
+	classes: {
+		bgColor: 'flatw-bg',
+		textColor: 'americano-text'
+	},
+	paragraph: 'Double shot of espresso with steamed milk.',
+}, {
+	title: 'Cappuccino',
+	id: 'cappuccino',
+	classes: {
+		bgColor: 'cappuccino-bg',
+		textColor: 'americano-text'
+	},
+	paragraph: 'Double shot of espresso topped with an airy thick layer of foamed milk.',
+}, {
+	title: 'Caffe latte',
+	id: 'latte',
+	classes: {
+		bgColor: 'latte-bg',
+		textColor: 'doppio-text'
+	},
+	paragraph: 'A shot of espresso in steamed milk lightly topped with foam.',
+}];
+const MENU = document.getElementById('mainMenu');
+const CUP = document.getElementById('coffeeCups');
+const ART = document.getElementById('articles');
+
+function render() {
+	document.getElementById('topHeader').innerHTML = `
+    <h1 class="mainHeader">Cat & C<span class="o"></span>ffee</h1>
+    `;
+	COFFEE.forEach(element => {
+		MENU.appendChild(listNode(element));
+		CUP.appendChild(cupNode(element));
+		ART.appendChild(contentNode(element));
+	});
+	document.getElementById('mainFooter').innerHTML = `
```

Comment:
The same with the footer - move it to the html

Reply:
:ok_hand: Done!